examples/bind_imported_app_to_offcluster_operator_managed_AWS_RDS_db/README.md

```diff
@@ -0,0 +1,307 @@
+# Binding an Imported app to an Off-cluster Operator Managed AWS RDS Database
+
+## Introduction
+
+This scenario illustrates binding an imported application to an off-cluster operated managed AWS RDS Database. The scenario also shows how to bind the backing service to two different independent applications and a use of the `envVarPrefix` feature of the Service Binding Operator to specify prefix for the names of the injected environment variables.
+
+## Actions to Perform by Users in 2 Roles
+
+In this example there are 2 roles:
+
+* Cluster Admin - Installs the operators to the cluster
+* Application Developer - Imports Shell and Node.js applications, creates a DB instance, creates requests to bind (connect) the applications and the DB.
+
+### Cluster Admin
+
+First, let's be the cluster admin. We need to install 2 operators into the cluster:
+
+* Service Binding Operator
+* Backing Service Operator
+
+The Backing Service Operator represents a database required by the
+applications. We'll use [aws-rds-operator](https://github.com/operator-backing-service-samples/aws-rds) to
+manage off-cluster RDS database instances on AWS.
+
+#### Install the Service Binding Operator using an `OperatorSource`
+
+Apply the [Service Binding Operator Source](./operator-source.service-binding-operator.yaml):
+
+``` shell
+make install-service-binding-operator-source
+```
+
+Then navigate to the `Operators`->`OperatorHub` in the OpenShift console under `openshift-marketplace` project and in the `Other` category select the `Service Bidning Operator` operator
```

Comment:
Very small nit - Should be "in the OpenShift console under the 'openshift-marketplace' project

Reply:
Thanks @ldimaggi... fixed.